tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
-    "lib": ["es6", "webworker"],
+    "target": "ES6",
```

Comment:
Remember how we had trouble jumping through hoops to get consuming packages to compile code from node_modules? I think that goes away if we target es5